_posts/2015-10-08-unacceptable-behaviors.md

```diff
@@ -4,8 +4,14 @@ title: Unacceptable behaviors
 date: 2015-10-08 00:08:00
 ---
 
-The product team is committed to providing a welcoming and safe environment for people of all races, gender identities, gender expressions, sexual orientations, physical abilities, physical appearances, socioeconomic backgrounds, nationalities, ages, religions, and beliefs. Discrimination and harassment are expressly prohibited in our employee handbook. Harassment includes, but is not limited to, intimidation; stalking; unwanted photography; inappropriate physical contact; use of sexual or discriminatory imagery, comments, or jokes; sexist, racist, ableist, or otherwise discriminatory or derogatory language; and unwelcome sexual attention. 
+The product team is committed to providing a welcoming and safe environment for people of all races, gender identities, gender expressions, sexual orientations, physical abilities, physical appearances, socioeconomic backgrounds, life experiences, nationalities, ages, religions, and beliefs. Discrimination and harassment are expressly prohibited in our employee handbook. Harassment includes, but is not limited to, intimidation; stalking; unwanted photography; inappropriate physical contact; use of sexual or discriminatory imagery, comments, or jokes; sexist, racist, ableist, or otherwise discriminatory or derogatory language; and unwelcome sexual attention.
```

Comment:
Does it make sense to modify `unwanted photography` to `unwanted recording and photography` to explicitly include video and audio recording?

Reply:
No objections to this change, but it should be broken out into a separate PR.

Reply:
I think it's a great suggestion and agree-- with such a minor change, I think we could probably get people to sign off on that much more quickly.